app/policies/draw_policy.rb

```diff
@@ -62,6 +62,14 @@ def start_lottery?
     edit? && record.pre_lottery?
   end
 
+  def oversubscription?
+    start_lottery?
+  end
+
+  def size_lock?
```

Comment:
Oh wait, I didn't do that since the same route is used to unlock the size, but admittedly size_lock is also confusing. We can use `toggle_size_lock` but it's just messy. What do you think?